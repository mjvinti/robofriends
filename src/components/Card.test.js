import React from 'react';
import { shallow } from 'enzyme';

import Card from './Card';

it('should render Card component', () => {
    expect(shallow(<Card id={1} name={'name'} email={'email@email.com'} />)).toMatchSnapshot();
});
