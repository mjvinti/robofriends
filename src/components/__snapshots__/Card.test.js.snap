// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Card component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/1?size=200x200"
  />
  <div>
    <h2>
      name
    </h2>
    <p>
      email@email.com
    </p>
  </div>
</div>
`;
